bundles/org.openhab.binding.openthermgateway/src/main/resources/OH-INF/thing/channels.xml

```diff
@@ -322,38 +342,378 @@
+		<item-type>Number:Dimensionless</item-type>
+		<label>Member ID</label>
+		<description>Member ID</description>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="vh_openthermversion">
+		<item-type>Number:Dimensionless</item-type>
+		<label>OpenTherm Version</label>
+		<description>OpenTherm version</description>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="vh_versiontype">
+		<item-type>Number:Dimensionless</item-type>
+		<label>Version Type</label>
+		<description>Version type</description>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="vh_relativeventilation">
+		<item-type>Number:Dimensionless</item-type>
+		<label>Relative Ventilation</label>
+		<description>Relative ventilation position</description>
+		<state readOnly="true" pattern="%d %%"/>
+	</channel-type>
+
+	<channel-type id="vh_relativehumidity">
+		<item-type>Number:Dimensionless</item-type>
+		<label>Relative Humidity</label>
+		<description>Relative humidity exhaust air</description>
+		<state readOnly="true" pattern="%d %%"/>
+	</channel-type>
+
+	<channel-type id="vh_co2level">
+		<item-type>Number:Dimensionless</item-type>
+		<label>CO2 Level</label>
+		<description>CO2 level exhaust air</description>
+		<state readOnly="true" pattern="%d ppm"/>
+	</channel-type>
+
+	<channel-type id="vh_supplyinlettemp">
+		<item-type>Number:Temperature</item-type>
+		<label>Supply Inlet Temperature</label>
+		<description>Supply inlet temperature</description>
+		<state readOnly="true" pattern="%.1f %unit%"/>
+	</channel-type>
+
+	<channel-type id="vh_supplyoutlettemp">
+		<item-type>Number:Temperature</item-type>
+		<label>Supply Outlet Temperature</label>
+		<description>Supply outlet temperature</description>
+		<state readOnly="true" pattern="%.1f %unit%"/>
+	</channel-type>
+
+	<channel-type id="vh_exhaustinlettemp">
+		<item-type>Number:Temperature</item-type>
+		<label>Exhaust Inlet Temperature</label>
+		<description>Exhaust inlet temperature</description>
+		<state readOnly="true" pattern="%.1f %unit%"/>
+	</channel-type>
+
+	<channel-type id="vh_exhaustoutlettemp">
+		<item-type>Number:Temperature</item-type>
+		<label>Exhaust Outlet Temperature</label>
+		<description>Exhaust outlet temperature</description>
+		<state readOnly="true" pattern="%.1f %unit%"/>
+	</channel-type>
+
+	<channel-type id="vh_actualexhaustfanspeed">
+		<item-type>Number:Dimensionless</item-type>
+		<label>Actual Exhaust Fan Speed</label>
+		<description>Actual exhaust fan speed</description>
+		<state readOnly="true" pattern="%d rpm"/>
+	</channel-type>
+
+	<channel-type id="vh_actualinletfanspeed">
+		<item-type>Number:Dimensionless</item-type>
+		<label>Actual Inlet Fan Speed</label>
+		<description>Actual inlet fan speed</description>
+		<state readOnly="true" pattern="%d rpm"/>
+	</channel-type>
+
+	<channel-type id="vh_nominalventenable">
+		<item-type>Switch</item-type>
+		<label>Nominal Ventilation Value Transfer</label>
```

Comment:
Labels are expected to be as short as possible. Guideline is 2-3 words with max 20-25 chars. See https://www.openhab.org/docs/developer/bindings/thing-xml.html#formatting-labels-and-descriptions

Reply:
Please see my comment on names derived form OT specification

Reply:
If they can't be shortend without loosing necessary information, I think it's ok to leave them as-is.